src/examples/index.js

```diff
@@ -4,10 +4,13 @@ import Sitemapper from '../assets/sitemapper';
 const exampleURL = 'https://www.walmart.com/sitemap_topic.xml';
 
 // Instantiate an instance
-const sitemapper = new Sitemapper({
+let sitemapper = new Sitemapper({
```

Comment:
@bsq-panagiotis I think this can actually stay a `const` since it's not being modified

Reply:
You are right